pxe/common_files/disk_imaging.sh

```
@@ -314,11 +339,253 @@ function modify_grub_cfg() {
 	fi
 }
 
+function get_part_num() {
+	dev=$1
+	is_nvme=$(echo $dev | grep nvme)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)